input/otlp/metrics_test.go

```diff
@@ -904,17 +915,61 @@ func TestConsumeMetricsWithOTelRemapper(t *testing.T) {
 				Name: "app",
 				Samples: []*modelpb.MetricsetSample{
 					{
-						Name:  "system.load.1",
-						Type:  modelpb.MetricType_METRIC_TYPE_GAUGE,
-						Value: 0.7,
+						Name:  "process.memory.usage",
+						Type:  modelpb.MetricType_METRIC_TYPE_COUNTER,
+						Value: 1024,
 					},
+				},
+			},
+		},
+		{
+			Service: &modelpb.Service{
+				Name:     "unknown",
+				Language: &modelpb.Language{Name: "unknown"},
+			},
+			Agent:     &modelpb.Agent{Name: "otlp", Version: "unknown"},
+			Timestamp: modelpb.FromTime(ts),
+			Metricset: &modelpb.Metricset{
+				Name: "app",
+				Samples: []*modelpb.MetricsetSample{
 					{
-						Name: "system.load.5",
```

Comment:
Question: Why is this here in the first place with no value, and why is it not present now after the PR?

Reply:
It is due to how otel lib behaves. There was a fix in v0.6.0 but that had to reverted in v0.6.1 so these lines need to be added back. I have added a comment explaining why it is there pasting below for reference:

```
				// TODO (lahsivjar): Opentelemetry lib currently adds all metrics, even
				// though the source metrics is not present. Due to this all the metrics
				// need to be asserted making the tests brittle. This should be fixed by
				// https://github.com/elastic/opentelemetry-lib/issues/6
```